Client/components/sidebarPics.jsx

```diff
@@ -11,33 +11,32 @@ const SidebarPics = (props) => {
     if (props.special.is) {
       return (
         <div>
-          <div style={{ height: '158px', border: '20px solid blue'}}>
+          <div style={{ height: '33.4%', width: '100%'}}  onClick={props.winUser}>
```

Comment:
Why styled-components previusly and in line styles here?
I feel these could be opportunities for more of those great component names with styled-components. Thoughts?